src/test/java/com/connexta/store/StoreITests.java

```diff
@@ -8,56 +8,68 @@
 
 import static com.connexta.store.controllers.StoreController.ADD_METADATA_URL_TEMPLATE;
 import static com.connexta.store.controllers.StoreController.CREATE_DATASET_URL_TEMPLATE;
+import static com.connexta.store.controllers.StoreController.METACARD_MEDIA_TYPE;
 import static com.connexta.store.controllers.StoreController.SUPPORTED_METADATA_TYPE;
-import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.not;
-import static org.hamcrest.core.AllOf.allOf;
-import static org.hamcrest.core.IsNull.notNullValue;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
✏️ Only import what we need.

Reply:
🤦‍♂

Reply:
I bumped up my settings in intellij -> shouldn't happen again